chicago/templates/chicago/about.html

```diff
@@ -7,18 +7,24 @@
         <br class="non-mobile-only"/>
         <h1 id='about'>About</h1>
 
-        <p><strong>Chicago Councilmatic</strong> tracks all things related to Chicago City Council: the <a href='/search/' target="_blank">legislation</a> introduced and passed, its various <a href='/committees/' target="_blank">committees</a> and the <a href='/events/' target="_blank">meetings</a> they hold, and the <a href='/council-members/' target="_blank">aldermen</a> themselves.</p>
+        <p><strong>Miami Commissionmatic</strong> tracks all things related to Miami-Dade County Commission: the <a
```

Comment:
Let's call this product "Miami-Dade Councilmatic" rather than "Miami Commissionmatic."